adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -64,7 +87,21 @@ class AkkaHttpAdapter private (private val options: AkkaHttpServerOptions)(impli
       queryExecution,
       requestInterceptor
     )
-    akkaInterpreter.toRoute(TapirAdapter.convertHttpEndpointToFuture(endpoint))
+    akkaInterpreter.toRoute(
+      convertHttpStreamingEndpoint(
+        endpoint.asInstanceOf[
```

Comment:
Shall we change the return type of `TapirAdapter.makeHttpUploadService` instead if casting?